taming/data/annotate_cub_dataset_mask.py

```diff
@@ -0,0 +1,46 @@
+import os
+import PIL
+from PIL import Image
+import glob
+import argparse
+from tqdm import tqdm
+
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description='List cub data files')
+    parser.add_argument('--image_dir', type=str, help='Path of images to be resized')
+    parser.add_argument('--output_dir', type=str, help='Path of resized images')
+    parser.add_argument('--segmentation_base_dir', type=str, help='Path of segmentation images')
+
+    args = parser.parse_args()
+    output_dir = '/fastscratch/mridul/cub_190_split_resized/official/CUB_200_2011/test_seg'
+
+
+    for folder in tqdm(sorted(glob.glob(os.path.join(args.image_dir, '*')))):
+        species = folder.split('/')[-1]
+        new_folder = os.path.join(args.output_dir, species)
+        if not os.path.isdir(new_folder):
+            os.makedirs(new_folder)
+        for image_path in sorted(glob.glob(folder+'/*')):
+            image_file = image_path.split('/')[-1]
+            segmentation_path = os.path.join(args.segmentation_base_dir, species, image_file)
+            segmentation_path = segmentation_path.replace('jpg', 'png')
+            try:
+                
+                image = PIL.Image.open(image_path)
```

Comment:
Maybe we should wait till you figure out what the gray areas in the dataset are?

Reply:
I tried looking for it, but there is no documentation for it anywhere about the grey areas.